src/foam/nanos/cron/CronSchedule.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.cron',
+  name: 'CronSchedule',
+  implements: [
+    'foam.nanos.cron.Schedule'
+  ],
+
+  javaImports: [
+    'foam.dao.DAO',
+    'foam.nanos.notification.Notification',
+    'foam.util.SafetyUtil',
+    'java.util.Calendar',
+    'java.util.concurrent.atomic.AtomicBoolean',
+    'java.util.Date'
+  ],
+
+  documentation: 'FOAM class that models a Cron script',
```

Comment:
This documentation looks wrong. Isn't this just a schedule, not a cron script?

Reply:
That's true. This documentation was taken from Cron.js as CronSchedule.js is just a carbon-copy of the schedule logic in Cron.js.

I didn't add documentation to the other schedules, so I'll add proper documentation for all of them now.